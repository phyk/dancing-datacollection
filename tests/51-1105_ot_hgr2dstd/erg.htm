<!DOCTYPE html PUBLIC "-//WAPFORUM//DTD XHTML Mobile 1.2//EN" "http://www.openmobilealliance.org/tech/DTD/xhtml-mobile12.dtd">
<HTML>
<HEAD>
  <META http-equiv="Content-Type" content="text/html; charset=utf-8">
  <META name="Author" content="Hessischer Tanzsportverband e.V.">
  <META name="GENERATOR" content="TopTurnierDigital">
  <meta http-equiv="Cache-Control" content="no-cache, no-store, must-revalidate" />
  <meta http-equiv="Pragma" content="no-cache" />
  <meta http-equiv="Expires" content="0" />
  <TITLE>11.05.2024 Hgr.II D Standard</TITLE>
  <link rel="stylesheet" type="text/css" href="topturnier.css">
  <meta name="viewport" content="width=device-width, initial-scale=1" />
</HEAD>
<body><div class="mainback">
<div class="eventhead">
<table border=0 width=100%>
<tr><td>11.05.2024 - OT, Hgr.II D Standard &#34;1&#34;</td><td width=30>
<a class="backbtn" href="index.htm" target="_top">&equiv;</a>
</td></tr>
</table>
</div>
<div class="maincontainer">
<div class="comphead">Ergebnis</div>
<hr class="line">
<div class="extract">
   <TABLE class="tab1">
    <TR>
     <TD class="td1" colspan="6">Endrunde</TD>
    </TR>
    <TR>
     <TD class="td2c" width="45px">Platz</TD>
     <TD class="td2">Paar/Club</TD>
     <TD class="td2cw" align=center nowrap>LW</TD>
     <TD class="td2cw" align=center nowrap>TG</TD>
     <TD class="td2cw" align=center nowrap>QU</TD>
     <TD class="td2" align=center nowrap>PZ</TD>
    </TR>
    <TR>
     <TD class="td3r">1.</TD>
     <TD class="td5">Jonathan Kummetz  /  Elisabeth Findei&szlig; (610)<BR><i>1. TC Rot-Gold Bayreuth</i></TD>
     <TD class="td3w">11131<BR><div class="pz">1.0</div></TD>
     <TD class="td3w">11141<BR><div class="pz">1.0</div></TD>
     <TD class="td3w">11131<BR><div class="pz">1.0</div></TD>
     <TD class="td3pz">&nbsp;<br> 3.0</TD>
    </TR>
    <TR>
     <TD class="td3r">2.</TD>
     <TD class="td5">Konstantin Pl&ouml;ger  /  Laura Utz (616)<BR><i>TSZ Blau-Gold Casino, Darmstadt</i></TD>
     <TD class="td3w">22215<BR><div class="pz">2.0</div></TD>
     <TD class="td3w">33253<BR><div class="pz">2.0</div></TD>
     <TD class="td3w">34244<BR><div class="pz">3.0</div></TD>
     <TD class="td3pz">&nbsp;<br> 7.0</TD>
    </TR>
    <TR>
     <TD class="td3r">3.</TD>
     <TD class="td5">Maik Rau  /  Carina Rau (617)<BR><i>Flensburger TC</i></TD>
     <TD class="td3w">43352<BR><div class="pz">3.0</div></TD>
     <TD class="td3w">56316<BR><div class="pz">6.0</div></TD>
     <TD class="td3w">53325<BR><div class="pz">2.0</div></TD>
     <TD class="td3pz">&nbsp;<br>11.0</TD>
    </TR>
    <TR>
     <TD class="td3r">4.</TD>
     <TD class="td5">Raphael Michel  /  Carolin Kimmig (611)<BR><i>TSC Gr&uuml;n-Gold Heidelberg</i></TD>
     <TD class="td3w">64643<BR><div class="pz">5.0</div></TD>
     <TD class="td3w">62632<BR><div class="pz">3.0</div></TD>
     <TD class="td3w">62452<BR><div class="pz">4.0</div></TD>
     <TD class="td3pz">&nbsp;<br>12.0</TD>
    </TR>
    <TR>
     <TD class="td3r">5.</TD>
     <TD class="td5">Sullivan Sadzik  /  Laura Mayer (619)<BR><i>TC Rot-Wei&szlig; Kaiserslautern</i></TD>
     <TD class="td3w">35426<BR><div class="pz">4.0</div></TD>
     <TD class="td3w">24425<BR><div class="pz">4.0</div></TD>
     <TD class="td3w">26516<BR><div class="pz">5.0</div></TD>
     <TD class="td3pz">&nbsp;<br>13.0</TD>
    </TR>
    <TR>
     <TD class="td3r">6.</TD>
     <TD class="td5">Emanuel Ostermaier  /  Katharina Dropmann (615)<BR><i>1. Tanzsport Zentrum Freising</i></TD>
     <TD class="td3w">56564<BR><div class="pz">6.0</div></TD>
     <TD class="td3w">45564<BR><div class="pz">5.0</div></TD>
     <TD class="td3w">45663<BR><div class="pz">6.0</div></TD>
     <TD class="td3pz">&nbsp;<br>17.0</TD>
    </TR>
   </TABLE>
   <TABLE class="tab2" border=0>
    <TR style="height: 5px; border:0;">
     <TD class="td5" colspan=3 style="height: 5px; border:0;"></TD>
    </TR>
    <TR>
     <TD class="td1" colspan=3>2. Zwischenrunde</TD>
    </TR>
    <TR>
     <TD class="td3r" width="45px">7.</TD>
     <TD class="td5">Thilo Schmid  /  Katharina Zierer (621)</TD>
     <TD class="td6">Dance Unlimited</TD>
    </TR>
    <TR>
     <TD class="td3r" width="45px">8.- 9.</TD>
     <TD class="td5">Tobias Knop  /  Cathrin Rube (607)</TD>
     <TD class="td6">TSC Rot-Gold Sinsheim</TD>
    </TR>
    <TR>
     <TD class="td3r" width="45px">8.- 9.</TD>
     <TD class="td5">Stefan M&uuml;hl  /  Eva Horlebein (613)</TD>
     <TD class="td6">TC Rot-Gold W&uuml;rzburg</TD>
    </TR>
    <TR>
     <TD class="td3r" width="45px">10.</TD>
     <TD class="td5">David Krause  /  Sophia Maier (609)</TD>
     <TD class="td6">TC Rot-Gold W&uuml;rzburg</TD>
    </TR>
    <TR>
     <TD class="td3r" width="45px">11.</TD>
     <TD class="td5">Simon Junski  /  Carolin Schilpp (606)</TD>
     <TD class="td6">TanzZentrum Ludwigshafen</TD>
    </TR>
    <TR>
     <TD class="td3r" width="45px">12.- 13.</TD>
     <TD class="td5">Felix Gasteiger  /  Nathalie Gleixner (604)</TD>
     <TD class="td6">Tanz-Club Laaber</TD>
    </TR>
    <TR>
     <TD class="td3r" width="45px">12.- 13.</TD>
     <TD class="td5">Linus Witascheck  /  Gilda Stechhan (624)</TD>
     <TD class="td6">Rot-Weiss-Klub Kassel</TD>
    </TR>
    <TR style="height: 5px; border:0;">
     <TD class="td5" colspan=3 style="height: 5px; border:0;"></TD>
    </TR>
    <TR>
     <TD class="td1" colspan=3>1. Zwischenrunde</TD>
    </TR>
    <TR>
     <TD class="td3r" width="45px">14.- 15.</TD>
     <TD class="td5">Jan Dingerkus  /  Diana Vorst (602)</TD>
     <TD class="td6">TSC Blau-Gold-Rondo Bonn</TD>
    </TR>
    <TR>
     <TD class="td3r" width="45px">14.- 15.</TD>
     <TD class="td5">Jonas Dreier  /  Johanna Grebe (603)</TD>
     <TD class="td6">Gie&szlig;ener Tanz-Club 74</TD>
    </TR>
    <TR>
     <TD class="td3r" width="45px">16.</TD>
     <TD class="td5">Georg Arndt  /  Anika Johlke (600)</TD>
     <TD class="td6">1. TSC Gr&uuml;n-Gold Leipzig 1947</TD>
    </TR>
    <TR>
     <TD class="td3r" width="45px">17.</TD>
     <TD class="td5">Sebastian Moch  /  Anna Melina Faude (612)</TD>
     <TD class="td6">TSC Residenz Ludwigsburg</TD>
    </TR>
    <TR>
     <TD class="td3r" width="45px">18.</TD>
     <TD class="td5">Waldemar Schilke  /  Isabell Grubert (620)</TD>
     <TD class="td6">TSC dancepoint, K&ouml;nigsbrunn</TD>
    </TR>
    <TR style="height: 5px; border:0;">
     <TD class="td5" colspan=3 style="height: 5px; border:0;"></TD>
    </TR>
    <TR>
     <TD class="td1" colspan=3>Vorrunde</TD>
    </TR>
    <TR>
     <TD class="td3r" width="45px">19.- 21.</TD>
     <TD class="td5">Marco Conrad  /  Alina Tempelmann (601)</TD>
     <TD class="td6">Tanzsportakademie Ludwigsburg</TD>
    </TR>
    <TR>
     <TD class="td3r" width="45px">19.- 21.</TD>
     <TD class="td5">Carsten Giersberg  /  Jennifer Rath (605)</TD>
     <TD class="td6">TSA d. TUS Stuttgart 1867</TD>
    </TR>
    <TR>
     <TD class="td3r" width="45px">19.- 21.</TD>
     <TD class="td5">Daniel Vitt  /  Tatjana Pankratz-Milstein (623)</TD>
     <TD class="td6">UTC M&uuml;nster</TD>
    </TR>
    <TR>
     <TD class="td3r" width="45px">22.- 23.</TD>
     <TD class="td5">Nicolas Koch  /  Christina Kalliafa (608)</TD>
     <TD class="td6">Tanzsportclub Solitude Kornwestheim</TD>
    </TR>
    <TR>
     <TD class="td3r" width="45px">22.- 23.</TD>
     <TD class="td5">Thomas R&ouml;sch  /  Ganna Kovtun (618)</TD>
     <TD class="td6">TSC Rot-Gold-Casino N&uuml;rnberg</TD>
    </TR>
    <TR>
     <TD class="td3r" width="45px">24.</TD>
     <TD class="td5">Lukas Th&uuml;rmer  /  Madeleine Klotzb&uuml;cher (622)</TD>
     <TD class="td6">TC Rot-Weiss Schw&auml;bisch Gm&uuml;nd</TD>
    </TR>
    <TR>
     <TD class="td3r" width="45px">25.</TD>
     <TD class="td5">Thorsten Olemotz  /  Jennifer Albach (614)</TD>
     <TD class="td6">Gie&szlig;ener Tanz-Club 74</TD>
    </TR>
   </TABLE>
<br>
</div>
</div>
</div>
  <P><FONT size="1" face="Arial">Diese Liste wurde mit <A TARGET="_blank" HREF="http://www.TopTurnier.de">TopTurnier f&uuml;r Windows V9.6b</A> erstellt.<br></FONT>
</body>
</HTML>
