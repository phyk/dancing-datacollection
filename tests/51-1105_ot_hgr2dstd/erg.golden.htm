<!DOCTYPE html>
<html>
 <head>
  <meta content="text/html; charset=utf-8" http-equiv="Content-Type"/>
  <title>
   11.05.2024 Hgr.II D Standard
  </title>
 </head>
 <body>
  <div>
   <div>
    <table>
     <tr>
      <td>
       11.05.2024 - OT, Hgr.II D Standard
      </td>
     </tr>
    </table>
   </div>
   <div>
    <div>
     Ergebnis
    </div>
    <hr/>
    <div>
     <table>
      <tr>
       <td>
        Endrunde
       </td>
      </tr>
      <tr>
       <td>
        Platz
       </td>
       <td>
        Paar/Club
       </td>
       <td>
        LW
       </td>
       <td>
        TG
       </td>
       <td>
        QU
       </td>
       <td>
        PZ
       </td>
      </tr>
      <tr>
       <td>
        1.
       </td>
       <td>
        Jonathan Kummetz / Elisabeth Findeiß (610)
        <br/>
        <i>
         1. TC Rot-Gold Bayreuth
        </i>
       </td>
       <td>
        11131
        <br/>
        <div>
         1.0
        </div>
       </td>
       <td>
        11141
        <br/>
        <div>
         1.0
        </div>
       </td>
       <td>
        11131
        <br/>
        <div>
         1.0
        </div>
       </td>
       <td>
        <br/>
        3.0
       </td>
      </tr>
      <tr>
       <td>
        2.
       </td>
       <td>
        Konstantin Plöger / Laura Utz (616)
        <br/>
        <i>
         TSZ Blau-Gold Casino, Darmstadt
        </i>
       </td>
       <td>
        22215
        <br/>
        <div>
         2.0
        </div>
       </td>
       <td>
        33253
        <br/>
        <div>
         2.0
        </div>
       </td>
       <td>
        34244
        <br/>
        <div>
         3.0
        </div>
       </td>
       <td>
        <br/>
        7.0
       </td>
      </tr>
      <tr>
       <td>
        3.
       </td>
       <td>
        Maik Rau / Carina Rau (617)
        <br/>
        <i>
         Flensburger TC
        </i>
       </td>
       <td>
        43352
        <br/>
        <div>
         3.0
        </div>
       </td>
       <td>
        56316
        <br/>
        <div>
         6.0
        </div>
       </td>
       <td>
        53325
        <br/>
        <div>
         2.0
        </div>
       </td>
       <td>
        <br/>
        11.0
       </td>
      </tr>
      <tr>
       <td>
        4.
       </td>
       <td>
        Raphael Michel / Carolin Kimmig (611)
        <br/>
        <i>
         TSC Grün-Gold Heidelberg
        </i>
       </td>
       <td>
        64643
        <br/>
        <div>
         5.0
        </div>
       </td>
       <td>
        62632
        <br/>
        <div>
         3.0
        </div>
       </td>
       <td>
        62452
        <br/>
        <div>
         4.0
        </div>
       </td>
       <td>
        <br/>
        12.0
       </td>
      </tr>
      <tr>
       <td>
        5.
       </td>
       <td>
        Sullivan Sadzik / Laura Mayer (619)
        <br/>
        <i>
         TC Rot-Weiß Kaiserslautern
        </i>
       </td>
       <td>
        35426
        <br/>
        <div>
         4.0
        </div>
       </td>
       <td>
        24425
        <br/>
        <div>
         4.0
        </div>
       </td>
       <td>
        26516
        <br/>
        <div>
         5.0
        </div>
       </td>
       <td>
        <br/>
        13.0
       </td>
      </tr>
      <tr>
       <td>
        6.
       </td>
       <td>
        Emanuel Ostermaier / Katharina Dropmann (615)
        <br/>
        <i>
         1. Tanzsport Zentrum Freising
        </i>
       </td>
       <td>
        56564
        <br/>
        <div>
         6.0
        </div>
       </td>
       <td>
        45564
        <br/>
        <div>
         5.0
        </div>
       </td>
       <td>
        45663
        <br/>
        <div>
         6.0
        </div>
       </td>
       <td>
        <br/>
        17.0
       </td>
      </tr>
     </table>
     <table>
      <tr>
       <td>
       </td>
      </tr>
      <tr>
       <td>
        2. Zwischenrunde
       </td>
      </tr>
      <tr>
       <td>
        7.
       </td>
       <td>
        Thilo Schmid / Katharina Zierer (621)
       </td>
       <td>
        Dance Unlimited
       </td>
      </tr>
      <tr>
       <td>
        8.- 9.
       </td>
       <td>
        Tobias Knop / Cathrin Rube (607)
       </td>
       <td>
        TSC Rot-Gold Sinsheim
       </td>
      </tr>
      <tr>
       <td>
        8.- 9.
       </td>
       <td>
        Stefan Mühl / Eva Horlebein (613)
       </td>
       <td>
        TC Rot-Gold Würzburg
       </td>
      </tr>
      <tr>
       <td>
        10.
       </td>
       <td>
        David Krause / Sophia Maier (609)
       </td>
       <td>
        TC Rot-Gold Würzburg
       </td>
      </tr>
      <tr>
       <td>
        11.
       </td>
       <td>
        Simon Junski / Carolin Schilpp (606)
       </td>
       <td>
        TanzZentrum Ludwigshafen
       </td>
      </tr>
      <tr>
       <td>
        12.- 13.
       </td>
       <td>
        Felix Gasteiger / Nathalie Gleixner (604)
       </td>
       <td>
        Tanz-Club Laaber
       </td>
      </tr>
      <tr>
       <td>
        12.- 13.
       </td>
       <td>
        Linus Witascheck / Gilda Stechhan (624)
       </td>
       <td>
        Rot-Weiss-Klub Kassel
       </td>
      </tr>
      <tr>
       <td>
       </td>
      </tr>
      <tr>
       <td>
        1. Zwischenrunde
       </td>
      </tr>
      <tr>
       <td>
        14.- 15.
       </td>
       <td>
        Jan Dingerkus / Diana Vorst (602)
       </td>
       <td>
        TSC Blau-Gold-Rondo Bonn
       </td>
      </tr>
      <tr>
       <td>
        14.- 15.
       </td>
       <td>
        Jonas Dreier / Johanna Grebe (603)
       </td>
       <td>
        Gießener Tanz-Club 74
       </td>
      </tr>
      <tr>
       <td>
        16.
       </td>
       <td>
        Georg Arndt / Anika Johlke (600)
       </td>
       <td>
        1. TSC Grün-Gold Leipzig 1947
       </td>
      </tr>
      <tr>
       <td>
        17.
       </td>
       <td>
        Sebastian Moch / Anna Melina Faude (612)
       </td>
       <td>
        TSC Residenz Ludwigsburg
       </td>
      </tr>
      <tr>
       <td>
        18.
       </td>
       <td>
        Waldemar Schilke / Isabell Grubert (620)
       </td>
       <td>
        TSC dancepoint, Königsbrunn
       </td>
      </tr>
      <tr>
       <td>
       </td>
      </tr>
      <tr>
       <td>
        Vorrunde
       </td>
      </tr>
      <tr>
       <td>
        19.- 21.
       </td>
       <td>
        Marco Conrad / Alina Tempelmann (601)
       </td>
       <td>
        Tanzsportakademie Ludwigsburg
       </td>
      </tr>
      <tr>
       <td>
        19.- 21.
       </td>
       <td>
        Carsten Giersberg / Jennifer Rath (605)
       </td>
       <td>
        TSA d. TUS Stuttgart 1867
       </td>
      </tr>
      <tr>
       <td>
        19.- 21.
       </td>
       <td>
        Daniel Vitt / Tatjana Pankratz-Milstein (623)
       </td>
       <td>
        UTC Münster
       </td>
      </tr>
      <tr>
       <td>
        22.- 23.
       </td>
       <td>
        Nicolas Koch / Christina Kalliafa (608)
       </td>
       <td>
        Tanzsportclub Solitude Kornwestheim
       </td>
      </tr>
      <tr>
       <td>
        22.- 23.
       </td>
       <td>
        Thomas Rösch / Ganna Kovtun (618)
       </td>
       <td>
        TSC Rot-Gold-Casino Nürnberg
       </td>
      </tr>
      <tr>
       <td>
        24.
       </td>
       <td>
        Lukas Thürmer / Madeleine Klotzbücher (622)
       </td>
       <td>
        TC Rot-Weiss Schwäbisch Gmünd
       </td>
      </tr>
      <tr>
       <td>
        25.
       </td>
       <td>
        Thorsten Olemotz / Jennifer Albach (614)
       </td>
       <td>
        Gießener Tanz-Club 74
       </td>
      </tr>
     </table>
     <br/>
    </div>
   </div>
  </div>
 </body>
</html>
